lib/src/commands/create.dart

```diff
@@ -151,8 +150,10 @@ class CreateCommand extends Command<int> {
     if (!isValidOrgName) {
       throw UsageException(
         '"$name" is not a valid org name.\n\n'
-        'A valid org name has 3 parts separated by "."'
-        'and only includes alphanumeric characters and underscores'
+        'A valid org name has:\n'
+        '  - At least 2 parts separated by "."\n'
+        '  - Each part must start with a letter\n'
+        '  - Only includes alphanumeric characters and underscores'
```

Comment:
According to the iOS spec:

> only alphanumeric characters (A-Z, a-z, and 0-9), hyphens (-), and periods (.)

I don't think we should allow underscores.

Reply:
That's a great idea, I'll update this